pkg/cmd/experimental/syncgroups/ad/augmented_ldapinterface_test.go

```diff
@@ -0,0 +1,115 @@
+package ad
+
+import (
+	"reflect"
+	"testing"
+
+	"gopkg.in/ldap.v2"
+
+	"github.com/openshift/origin/pkg/auth/ldaputil"
+	"github.com/openshift/origin/pkg/auth/ldaputil/testclient"
+)
+
+const (
```

Comment:
These shouldn't exist.  They're letting you avoid creating composable methods and its making your tests hard to read.  You've done this in all three it looks like.